system/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "node-function-buildpack",
+  "version": "0.4.1",
+  "description": "A buildpack to build node functions",
+  "dependencies": {
+    "@projectriff/message": "^1.0.0",
+    "cloudevents-sdk": "^0.3.0"
```

Comment:
do we really want `^0.3.0` or just `0.3.0`

Reply:
@joshwlewis we actually aren't parsing the cloudevent right now, maybe we can remove this here and the lock file for now.